Sources/SwiftlyCore/SwiftlyCore.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-public let version = SwiftlyVersion(major: 0, minor: 3, patch: 0)
+public let version = SwiftlyVersion(major: 0, minor: 4, patch: 0, suffix: "dev")
```

Comment:
Is it expected to include a prefix on v0.x releases? I've only ever used a prefix when I have a beta/alpha/rc for a major version.

Reply:
I don't think that there's anything in [semver](https://semver.org) that precludes pre-releases from being used in 0.x.